rust_dev_preview/iam/README.md

```diff
@@ -1,58 +1,72 @@
-# AWS SDK for Rust code examples for IAM
+# IAM code examples for AWS SDK for Rust
 
-## Purpose
+## Overview
 
-These examples demonstrate how to perform several AWS Identity and Access Management (IAM) operations using the developer preview version of the AWS SDK for Rust.
+Shows how to use the AWS SDK for Rust to manage Amazon IAM resources.
 
-IAM is a web service for securely controlling access to AWS services.
+*IAM is a web service for securely controlling access to AWS services. With IAM, you can centrally manage users,
+security credentials such as access keys, and permissions that control which AWS resources users and applications can
+access.*
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform
+  the task. For more information, see
+  [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information,
+  see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
 
 ## Code examples
 
-- [Create a role](src/bin/create-role.rs) (CreateRole)
+### Single action
 
-## ⚠ Important
+* [Listing Roles](src/iam-service-lib.rs) (ListRoles)
+* [Listing Users](src/iam-service-lib.rs) (ListUsers)
+* [Listing Groups](src/iam-service-lib.rs) (ListGroups)
+* [Attaching Role Policies](src/iam-service-lib.rs) (AttachRolePolicy)
+* [Getting an Account Password Policy](src/iam-service-lib.rs) (GetAccountPasswordPolicy)
+* [Listing SAML Providers](src/iam-service-lib.rs) (ListSAMLProviders)
+* [Listing Role Policies](src/iam-service-lib.rs) (ListRolePolicies)
+* [Getting a Role](src/iam-service-lib.rs) (GetRole)
+* [Listing Policies](src/iam-service-lib.rs) (ListPolicies)
+* [Creating a Service Linked Role](src/iam-service-lib.rs) (CreateServiceLinkedRole)
+* [Creating a Role](src/iam-service-lib.rs) (CreateRole)
+* [Listing Attached Role Policies](src/iam-service-lib.rs) (ListAttachedRolePolicies)
+* [Getting a Policy](src/iam-service-lib.rs) (GetPolicy)
```

Comment:
Use sentence case for titles. For example: 
Listing roles
Listing users
Listing SAML providers

Reply:
fixed